db/migrate/20230609140544_create_post.rb

```diff
@@ -5,9 +5,9 @@ def change
     create_table :posts do |t|
       t.string :title
       t.string :text
-      t.integer :comment_counter
+      t.integer :comments_counter
       t.integer :likes_counter
-
+      add_reference :posts, :author, column: :author_id, null: false, foreign_key: { to_table: :users }
```

Comment:
- [ ] Kindly move this piece of code outside the `create_table :posts` block, let the code be between `line12 & line13`
![Screenshot from 2023-06-10 11-38-29](https://github.com/mudasiri/mybloggerapp/assets/46388113/d5f03e15-eb63-4491-b04d-edef0909adbc)